Resources/doc/basic-usage.rst

```diff
@@ -264,4 +264,20 @@ If browser supports WebP, the request ``https://localhost/media/cache/resolve/th
 will be redirected to ``https://localhost/media/cache/thumbnail_web_path/images/cats.jpeg.webp``
 otherwise to ``https://localhost/media/cache/thumbnail_web_path/images/cats.jpeg``
 
+.. note::
+
+    Using an unsecured connection (HTTPS) on your site can cause problems with
```

Comment:
`unsecured connection (HTTPS)` this does not seem to be right. Unsecured is not HTTPS

Reply:
Yes.  Exactly.  It seemed to me that writing `unsecured connection (HTTP)` would not be clear enough. Therefore, i'm talking about HTTPS, hinting at the security of the connection. Perhaps it would be better to write `unsecured connection (non HTTPS)`.

Reply:
can you send a new PR?

Reply:
@lsmith77 yes of course #1340